### ISSUE TITLE
New problems has occured with the latest winetricks and wine itself.

### ISSUE THREAD

**Issue by nolsen42** [roblox-linux-wrapper/roblox-linux-wrapper#180]
When trying to install, I get this

```fixes/roblox-wine/dosdevices/c:/windows/temp/_winhttp/i386/new/winhttp.dl_
Using native,builtin override for following DLLs: winhttp
Executing /usr/bin/wine regedit C:\windows\Temp\_winhttp\override-dll.reg
Using winetricks 20180513-next - sha256sum: c5438329b437e02416de6278346b5d0a8cb3d586415d069288fc85a300c9a02e with wine-3.8 (Staging) and WINEARCH=win32
Executing w_do_call d3dx9_43
Executing load_d3dx9_43 
Executing cabextract -q -d /home/nolsen/.local/share/wineprefixes/roblox-wine/dosdevices/c:/windows/temp/_d3dx9_43 -L -F *d3dx9_43*x86* /home/nolsen/.cache/winetricks/directx9/directx_Jun2010_redist.exe
Executing cabextract -q -d /home/nolsen/.local/share/wineprefixes/roblox-wine/dosdevices/c:/windows/system32 -L -F d3dx9_43.dll /home/nolsen/.local/share/wineprefixes/roblox-wine/dosdevices/c:/windows/temp/_d3dx9_43/*.cab
/home/nolsen/.local/share/wineprefixes/roblox-wine/dosdevices/c:/windows/temp/_d3dx9_43/*.cab: No such file or directory
------------------------------------------------------
Note: command cabextract -q -d /home/nolsen/.local/share/wineprefixes/roblox-wine/dosdevices/c:/windows/system32 -L -F d3dx9_43.dll /home/nolsen/.local/share/wineprefixes/roblox-wine/dosdevices/c:/windows/temp/_d3dx9_43/*.cab returned status 1. Aborting.
```

**Comment by nolsen42** [comment]
Apparently, the cache needs to be cleared, but I also have to point out that something breaks with the latest version of winetricks.

```Unknown arg sound=alsa
Usage: /usr/local/bin/winetricks [options] [command|verb|path-to-verb] ...
Executes given verbs.  Each verb installs an application or changes a setting.

Options:
    --country=CC      Set country code to CC and don't detect your IP address
    --force           Don't check whether packages were already installed
    --gui             Show gui diagnostics even when driven by commandline
    --isolate         Install each app or game in its own bottle (WINEPREFIX)
    --self-update     Update this application to the last version
    --update-rollback Rollback the last self update
-k, --keep_isos       Cache isos (allows later installation without disc)
    --no-clean        Don't delete temp directories (useful during debugging)
-q, --unattended      Don't ask any questions, just install automatically
-r, --ddrescue        Retry hard when caching scratched discs
    --showbroken      Even show verbs that are currently broken in wine
-t  --torify          Run downloads under torify, if available
    --verify          Run (automated) GUI tests for verbs, if available
-v, --verbose         Echo all commands as they are executed
-h, --help            Display this message and exit
-V, --version         Display version and exit

Commands:
list                  list categories
list-all              list all categories and their verbs
apps list             list verbs in category 'applications'
benchmarks list       list verbs in category 'benchmarks'
dlls list             list verbs in category 'dlls'
games list            list verbs in category 'games'
settings list         list verbs in category 'settings'
list-cached           list cached-and-ready-to-install verbs
list-download         list verbs which download automatically
list-manual-download  list verbs which download with some help from the user
list-installed        list already-installed verbs
```

**Comment by nolsen42** [comment]
To further complicate things, now Roblox is claiming that "it cannot connect to the internet", despite working fine if you use the standard 64arch wineprefix, or anything that's not RLW.

**Comment by CollinScripter** [comment]
I've also been receiving errors on connecting to the ROBLOX website
Occurs both on wine-devel and wine-stable
+1

**Comment by alfonsojon** [comment]
This should be fixed with today's release. It no longer depends on `wininet`; in fact, I've removed all winetricks as they are no longer necessary.

**Comment by alfonsojon** [comment]
confirmed fixed on latest winetricks and wine-staging 5.4
